libiocage/cli/rename.py

```diff
@@ -39,7 +39,7 @@ def cli(ctx, jail, name):
     """
 
     logger = ctx.parent.logger
-    try :
+    try:
```

Comment:
while you're touching these functions, you might as well add `-> None` to the the functions (and perhaps _other_ type signatures, too :)